Core/Object Arts/Dolphin/Base/DolphinTest.cls

```diff
@@ -77,6 +77,14 @@ runKnownBug: aString test: aSymbol
 	#todo.	"Make this controllable by an option or class var"
 	^false!
 
+setUp
+
```

Comment:
I'm happy to take it out, but I'd point out that there isn't anything UI-specific about allowing background tasks to complete between tests. In particular, a test that forks a process and then expect it to complete running within a number of milliseconds will behave differently if it is run as a suite and if it is run in isolation. Further, when running non-UI tests from the TestRunner, it is frustrating to get no feedback. Finally, if there are no pending tasks/messages, then the overhead of checking should be minimal.